tests/helpers/perf_run_helper.rb

```diff
@@ -29,6 +35,14 @@ module PerfRunHelper
   METRIC_RESULTS_DIR = "/root/gatling-puppet-load-test/simulation-runner/results"
   CURRENT_TUNE_SETTINGS = "current_tune_settings.json"
 
+  def logger
+    @logger ||= Beaker::Logger.new
+  end
+
+  def current_test_name
```

Comment:
Explain why this is here!
Two contexts:
* Run without beaker (uses given test_type from CLI)
* Run with beaker (should inherited [beaker method](https://github.com/puppetlabs/beaker/blob/a01b5732a7d6cec146bd18d08795c64aafe1d2f1/lib/beaker/dsl/helpers/test_helpers.rb#L12), but cannot get to it).  This is a hack to re-implement the method to solve the problem.  This may be solved by calling "extend" rather than "include" on the beaker dsl at the end of this file.